core/src/main/scala/akka/persistence/jdbc/state/scaladsl/JdbcDurableStateStoreQuery.scala

```diff
@@ -0,0 +1,79 @@
+package akka.persistence.jdbc.state.scaladsl
+
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.util.Try
+import akka.NotUsed
+import akka.persistence.query.scaladsl.DurableStateStoreQuery
+import akka.persistence.state.scaladsl.GetObjectResult
+import akka.persistence.query.{ Offset, Sequence => APSequence, NoOffset }
+import akka.persistence.query.DurableStateChange
+import akka.persistence.jdbc.config.DurableStateTableConfiguration
+import akka.persistence.jdbc.state.{ AkkaSerialization, DurableStateQueries, DurableStateTables }
+import akka.persistence.jdbc.journal.dao.FlowControl
+import akka.serialization.Serialization
+import akka.stream.Materializer
+import akka.stream.scaladsl.{ Sink, Source }
+import slick.jdbc.{ JdbcBackend, JdbcProfile }
+
+class JdbcDurableStateStoreQuery[A](
+    stateStore: JdbcDurableStateStore[A],
+    db: JdbcBackend#Database,
+    profile: JdbcProfile,
+    durableStateConfig: DurableStateTableConfiguration,
+    serialization: Serialization)(implicit m: Materializer, e: ExecutionContext)
+    extends DurableStateStoreQuery[A] {
```

Comment:
I think we can have one implementation that `extends DurableStateUpdateStore with DurableStateStoreQuery`, i.e. these addition can be added to the `JdbcDurableStateStore` class instead.

Reply:
done ..